uber/models/mivs.py

```diff
@@ -423,6 +423,34 @@ def confirmed(self):
             and self.studio \
             and self.studio.group_id
 
+    @property
+    def has_been_accepted(self):
```

Comment:
Here's a breakdown of our MIVS export:
We only return games that have been accepted (the `has_been_accepted` property is defined here mostly so I can access the same function on both MIVS games and MITS games).

Name: The game's studio's name.
Subtitle: The game's own name.
Description: The game's description.
Location: Nothing, but we could conceivably return MIVS' location here.
Image: The first image we find that the team has marked as one of this game's best screenshots.
Thumbnail: The second image we find that the team has marked as one of this game's best screenshots. I'll need to edit this to return just the first image if there aren't two, though I think this shouldn't be possible.